git/homework.py

```diff
@@ -127,7 +135,7 @@ def alphabet() -> dict:
         alphabet()
         >>> {"a": 1, "b": 2 ...}
     """
-    pass
+    return dict(OrderedDict((i+1,k) for i, k in enumerate(ascii_lowercase)))
```

Comment:
We can make it easier.
Try this: `dict(enumerate(ascii_lowercase, 1))`

Reply:
![image](https://user-images.githubusercontent.com/51755199/65627700-92c7ac00-dfd8-11e9-8d22-6e0bfe9d878f.png)

Reply:
Seems like ok. Both of the changes proposed.

Reply:
I have deleted 1 unused import & change Value Error message code.